pkgs/development/python-modules/google-cloud-iam/default.nix

```diff
@@ -12,11 +12,12 @@
 
 buildPythonPackage rec {
   pname = "google-cloud-iam";
-  version = "2.2.0";
+  version = "2.3.1";
+  disabled = isPy27;
```

Comment:
Please use pythonOlder instead. I am sure that probably python3.5/3.6 is not supported.

Reply:
Done